### PR TITLE
Adds a slight warning that you may have screwed up to the nuclear bomb

### PR DESCRIPTION
**Pull request by Qwertytoforty**
## What Does This PR Do

Adds a message to a person attempting to put a plant gene disk into the nuke that they are using a wrong disk.

## Why It's Good For The Game

A funny realization to the nuclear operative that looted the captain that they looted the wrong disk.

This doesn't ruin the prank either, nuke ops should never do this as they have a pinpointer, which would keep most from not falling for the prank, and if they use the wrong disk currently, it just bounces off. This just makes it a little funnier.

## Changelog
:cl:
add: Adds a message for when someone tries to put a plant disk into the nuke.
/:cl:

### CODE REVIEW

File: code/game/gamemodes/nuclear/nuclearbomb.dm [hunk 1]
```diff
@@ -68,6 +68,9 @@ GLOBAL_VAR(bomb_set)
 		else
 			to_chat(user, "<span class='notice'>You need to deploy [src] first.</span>")
 		return
+	else if(istype(O, /obj/item/disk/plantgene))
+		to_chat(user, "<span class='warning'> You try to put the Nuclear Authentication Disk into the nuke, but it does not fi- Wait, this isn't the right disk!</span>")
```
Comment (Kyep) [inline code/game/gamemodes/nuclear/nuclearbomb.dm:72]:
```suggestion
		to_chat(user, "<span class='warning'>You try to plant the disk in the nuke, but after some rooting around, it just doesn't fit. You branch out, and read the printed instructions.. it turned out you've been bamboo-zled, this isn't a nuclear disk at all!</span>")
```

File: code/game/gamemodes/nuclear/nuclearbomb.dm [hunk 2]
```diff
@@ -68,6 +68,9 @@ GLOBAL_VAR(bomb_set)
 		else
 			to_chat(user, "<span class='notice'>You need to deploy [src] first.</span>")
 		return
+	else if(istype(O, /obj/item/disk/plantgene))
+		to_chat(user, "<span class='warning'>You try to plant the disk, but despite rooting around, it won't fit! After you branch out to read the instructions, you find out where the problem stems from. You've been bamboo-zled, this isn't a nuclear disk at all!</span>")
```
Comment (AffectedArc07) [inline code/game/gamemodes/nuclear/nuclearbomb.dm:72]: I want to absolutely cry at this pun, but I guess it is **pun**ishment for those who make this mistake. Not an objection, but good god this hurts me.

*I only realised that pun above was a thing as I was typing this out. Its 1AM and I need help*

Comment (Qwertytoforty) [reply code/game/gamemodes/nuclear/nuclearbomb.dm:72]: May have to visit the branch manager that suggested it

**Comment by moxian** [comment]
Do you maybe want to add a similar warning for `/obj/item/disk/tech_disk` and/or `/obj/item/disk/design_disk` while at it?

**Comment by AffectedArc07** [comment]
> Do you maybe want to add a similar warning for `/obj/item/disk/tech_disk` and/or `/obj/item/disk/design_disk` while at it?

Those dont look anything like the nuke disk. The main issue is people look for the green disk, and the plant ones look almost the same.

**Review by Kyep — APPROVED** [review]

**Review by AffectedArc07 — APPROVED** [review]

**Pull request merged**